lib/common.js

```diff
@@ -1777,8 +1777,9 @@ module.exports = {
         function setup(req, res, next) {
             // General request setup
             req.config = options;
-            req.moray = clients.moray;
-            req.boray = clients.boray;
+            // req.moray = clients.moray;
```

Comment:
Looks like these could possibly be removed.

Reply:
I removed those two lines.